lib/mergent.rb

```diff
@@ -5,7 +5,11 @@
 require_relative "mergent/version"
 
 module Mergent
+  ENDPOINT = "https://api.mergent.co/v1"
+
   class << self
-    attr_accessor :api_key
+    attr_accessor :api_key, :endpoint
```

Comment:
I didn't document this, because it's really only useful for people working on Mergent itself. I can add docs of course.